tests/e2e/e2e_test.go

```diff
@@ -4,64 +4,76 @@ import (
 	"fmt"
 	"strconv"
 	"time"
+
+	"github.com/osmosis-labs/osmosis/v7/tests/e2e/chain"
 )
 
-func (s *IntegrationTestSuite) TestSuperfluidVoting() {
+func (s *IntegrationTestSuite) TestIBCTokenTransfer() {
 	chainA := s.chainConfigs[0].chain
-	s.Run("superfluid_vote_chainA", func() {
-		s.submitSuperfluidProposal(chainA, "gamm/pool/1")
-		s.depositProposal(chainA)
-		s.voteProposal(chainA, s.chainConfigs[0])
-		// send gamm tokens to validator's other wallet (non self-delegation wallet)
-		s.sendTx(chainA, 0, "100000000000000000000gamm/pool/1", chainA.Validators[0].PublicAddress, chainA.Validators[0].PublicAddress2)
-		// lock tokens from validator 0 on chain A
-		s.lockTokens(chainA, 0, "100000000000000000000gamm/pool/1", "240s", "val2")
-		// superfluid delegate from validator 0 non self-delegation wallet to validator 1 on chain A
-		s.superfluidDelegate(chainA, s.chainConfigs[0].chain.Validators[1].OperAddress, "val2")
-		// create a text prop, deposit and vote yes
-		s.submitTextProposal(chainA, "superfluid vote overwrite test")
-		s.depositProposal(chainA)
-		s.voteProposal(chainA, s.chainConfigs[0])
-		// set delegator vote to no
-		s.voteNoProposal(chainA, 0, "val2")
+	chainB := s.chainConfigs[1].chain
+
+	// compare coins of receiver pre and post IBC send
+	// diff should only be the amount sent
+	s.sendIBC(chainA, chainB, chainB.Validators[0].PublicAddress, chain.OsmoToken)
+}
+
+func (s *IntegrationTestSuite) TestSuperfluidVoting() {
+	chainA := s.chainConfigs[0]
+	s.submitSuperfluidProposal(chainA, "gamm/pool/1")
+	s.depositProposal(chainA)
+	s.voteProposal(chainA)
+	s.ExecTx(chainA.chain.ChainMeta.Id, 0, []string{"chmod", "+x", "./add_account.sh"}, "")
```

Comment:
How much more difficult is it to execute this transaction via RPC? I'm afraid this is not very extensive adding files for transaction

Reply:
Do you know how to sign a tx with a key that isn't on your keyring? I spend some hours trying to figure out how to add it and I just kept running into problems

Reply:
Since we are using a test keyring backend, it is storing keys insecurely on a disk. Since we are mounting a volume, it should be accessible on the host, should it not?

Reply:
https://github.com/osmosis-labs/cosmos-sdk/blob/889979c0e22a744cd1e4de82140d915863aca219/crypto/keyring/doc.go#L42

Reply:
Right but the only way I have been able to figure it out is via bash "osmosisd keys add NAME --recover" and haven't been able to get the key added through just golang (I'm positive its possible, but my lack of experience with it has made it challenging)

Reply:
No worries, I'm not sure what the best way is either - just trying to brainstorm. 

After looking more into it - I don't think there is an RPC for that.

We could use some `testutil` functions directly in the test runner to create an account: https://github.com/osmosis-labs/cosmos-sdk/blob/0b8e530add916d724f4ea077bb174c275861647b/testutil/key.go#L31

or even import the `NewKeyring(...)` and `NewAccount(...)` functions directly: https://github.com/osmosis-labs/cosmos-sdk/blob/0b8e530add916d724f4ea077bb174c275861647b/crypto/keyring/keyring.go#L77

However, we've been trying to avoid depending on the codebase directly in tests so these are not the best way to go forward.

I think what we discussed offline is best - try to avoid piping the mnemonic and use an auto-generated one so that we can execute the command directly via dockertest

Reply:
Ended up going with your suggestion of simply adding a new random key since it didn't need to have a balance initially (had to still use regex expression sadly to pull the address out). Thanks for the suggestion!